modules/project-factory/factory-projects.tf

```diff
@@ -20,7 +20,7 @@ locals {
   _hierarchy_projects = (
     {
       for f in try(fileset(local._folders_path, "**/*.yaml"), []) :
-      basename(trimsuffix(f, ".yaml")) => merge(
+      replace(trimsuffix(f, ".yaml"), "/", "-") => merge(
```

Comment:
do you really need the replace? I think you could keep '/' in the keys.

Reply:
Well yes if you want to use the filename as project id. 

Ignore

Reply:
Same comment I made above, path should be left intact